content/applications/marketing/marketing_automation/getting_started/workflow_activities.rst

```diff
@@ -1,80 +1,251 @@
-:guilabel:`Add New Activity`. Doing so reveals a :guilabel:`Create Activities` pop-up window. This
-pop-up window is a blank activity template, where specific parameters can be set for that particular
-activity.
+Add workflow activities
+=======================
 
-The following fields are available in the :guilabel:`Create Activities` pop-up window (when
-:guilabel:`Add New Activity` is clicked):
+To add workflow activities to a marketing campaign, navigate to the bottom of a pre-existing or new
+campaign detail form, beneath the target audience configuration fields, and click :guilabel:`Add new
+activity`.
+
+Doing so reveals a :guilabel:`Create Activities` pop-up window. This pop-up window is a blank
+activity template, where specific parameters can be set for that particular activity.
 
 .. image:: workflow_activities/activity-template.png
    :align: center
-   :alt: An activity template in Odoo Marketing Automation.
-
-- :guilabel:`Activity Name`: the title of the activity.
-- :guilabel:`Activity Type`: choose between `Email`, `Server Action` (internal Odoo operation), or
-  `SMS`.
-- :guilabel:`Mail Template`: choose from pre-configured templates (or create a new one on-the-fly).
-- :guilabel:`Trigger`: choose when this activity should be triggered.
-- :guilabel:`Expiry Duration`: configure to stop the actions after a specific amount of time (after
-  the scheduled date). When selected, a :guilabel:`Cancel after` field appears, in which the user
-  can choose how many :guilabel:`Hours, Days, Weeks, or Months` they want the actions to cease after
-  the initial date.
-- :guilabel:`Activity Filter`: domain related to this activity (and all subsequent child
-  activities).
-- :guilabel:`Applied Filter`: activity will *only* be performed if it satisfies the specified domain
-  (filter).
-
-After the activity's settings are fully configured, click :guilabel:`Save & Close` to save the
-activity and return to the marketing automation campaign form, :guilabel:`Save & New` to save the
-activity and immediately create another one in a fresh :guilabel:`Create Activities` pop-up window,
-or :guilabel:`Discard` to delete the activity and return to the marketing automation campaign form.
+   :alt: A workflow activity template pop-up window in Odoo Marketing Automation.
+
+First, enter a name for the activity in the :guilabel:`Activity Name` field, and proceed to
+configure the following options. Once ready, either click: :guilabel:`Save & Close` (to save the
+activity and close the pop-up form), :guilabel:`Save & New` (to save the activity and instantly
+create an additional activity on a fresh :guilabel:`Create Activities` pop-up form), or
+:guilabel:`Discard` to delete the activity.
+
+Activity types
+--------------
+
+Then, select the :guilabel:`Activity Type`. Choose between :guilabel:`Email`, :guilabel:`Server
+Action` (an internal action within the database), or :guilabel:`SMS`.
```

Comment:
Is there a doc about server actions we could link to here, just so the curious-minded could read more about them? Not totally necessary, but would be a nice-to-have

Reply:
Oh, great idea. Unfortunately, I don't believe there is any specific doc about server actions. But I fully understand where something like that would come in handy in this instance. So, if anything like that *does* exist, I'll be sure to add it in here -- for sure! Thanks for the suggestion! 👍